views-react/src/main/java/io/micronaut/views/react/JSContext.java

```diff
@@ -0,0 +1,146 @@
+package io.micronaut.views.react;
+
+import jakarta.annotation.PostConstruct;
+import jakarta.annotation.PreDestroy;
+import jakarta.inject.Inject;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+class JSContext implements AutoCloseable {
```

Comment:
provide a scope for this context. If a new one is needed each time add `Prototype` otherwise `Singleton`

Reply:
Contexts are pooled, so I'm not sure their lifetime maps to anything the DI container can handle.